src/datasets/vtr_dataset.py

```diff
@@ -70,7 +70,13 @@ def __init__(
         ratio: float = 0.7,
     ):
         logger.info(f"Initializing VTRDatasetOCR with {len(labeled_texts)} samples, use max seq len {max_seq_len}")
-        self.labeled_texts = labeled_texts
+
+        self.labeled_texts = []
+        self.char_set: set = set()
+        for sample in labeled_texts:
+            self.labeled_texts.append({"text": clean_text(sample["text"]), "label": sample["label"]})
```

Comment:
I suggest checking if the cleaned text is empty and skipping such examples. At the end of init, you can log how many samples you got from the data.

Reply:
Also, I think using 2 lists for texts and labels would be more convenient for future using